plugin.xml

```diff
@@ -100,6 +100,7 @@
             <clobbers target="CameraPopoverHandle" />
          </js-module>
 
+         <resource-file src="src/ios/Camera.strings" target="en.lproj/Camera.strings" />
```

Comment:
I don't know anything about iOS. What does `target="en.lproj/Camera.strings"` mean here?

Reply:
It will copy the hardcoded message into a language file named Camera.strings and we can load the message from this file

Reply:
Ok, so `en.lproj` is a magic thing that the project will know about and load `en.lproj/Camera.strings` without any additional configuration?

Should the file maybe be called `cordova-plugin-camera.strings`?

Reply:
The en.lproj or fr.lproj or ar.lproj etc are understood by app to load corresponding language files
You are free to name the strings file just update the plugin.xml accordingly

Reply:
That was really more a question - to make sure developers know what the file is from and for. You created the PR, so you can decide and adapt or not.